docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -210,7 +212,7 @@ To start a Keycloak Server you can use Docker and just run the following command
 docker run --name keycloak -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -p 8543:8443 -v "$(pwd)"/config/keycloak-keystore.jks:/etc/keycloak-keystore.jks quay.io/keycloak/keycloak:{keycloak.version} start  --hostname-strict=false --https-key-store-file=/etc/keycloak-keystore.jks
 ----
 
-where `keycloak.version` should be set to `17.0.0` or higher.
+where `keycloak.version` should be set to `23.0.0` or higher and the `keycloak-keystore.jks` can be found in https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-keycloak-authorization-quickstart/config/keycloak-keystore.jks[quarkus-quickstarts/security-keycloak-authorization-quickstart/config]
```

Comment:
```suggestion
where `keycloak.version` should be set to `23.0.0` or higher.
```

Users are not expected to use it somehow, the command above picks it up with `$(pwd)"/config/keycloak-keystore.jks`, so it is not clear what adding this information helps with, alongside a more important KC version update info

Reply:
My point in here is that if user not using quickstart but go through this by him self. He can easily miss this. I can drop it if you stand on this.